backend/src/routes/ansibletower.ts

```diff
@@ -13,6 +13,9 @@ interface AnsibleCredential {
   token: string
 }
 
+// must match ansiblePaths in frontend/src/resources/utils/resource-request.ts
+const ansiblePaths = ['/api/v2/job_templates/', '/api/v2/workflow_job_templates/']
```

Comment:
Please be aware that once this PR is merged, backend changes will be required to access new Ansible Tower APIs @RobDolares. (Related to [ACM-3842](https://issues.redhat.com/browse/ACM-3842))